### DISCUSSION TITLE
Activation of DeepSource

### DISCUSSION THREAD

**Discussion by HarshCasper** [openstates/issues#381] [General] [answered]
Hi 👋 

One of my Pull Requests around fixing Code Quality Issues with DeepSource was merged here: https://github.com/openstates/openstates-scrapers/pull/3636 

I'd just like to inform you that the issues fixed here were detected by running DeepSource analysis on the repo. If you like, you can activate analysis for your repository to detect such code quality issues/bug risks on the fly for every change made. You can also use the Autofix feature to fix them with one click.

The `.deepsource.toml` file you merged will only take effect if you activate analysis for this repo.

Here's what you can do if you wish to activate DeepSource to continuously analyze your repository:

- [Sign up](https://deepsource.io/signup) on DeepSource and activate analysis for this repository.
- Create `.deepsource.toml` configuration which you can use to configure your analysis settings (My PR already added that, but feel free to edit it anytime).
- Track/Check analysis [here](http://deepsource.io/gh/openstates/openstates-scrapers/).

If you have any doubts or questions, you can check out the [docs](https://deepsource.io/docs/), or feel free to reach out :)

**Comment by jamesturk** [comment, marked as answer]
thanks, giving it a try :)
